src/kw_config_loader.sh

```diff
@@ -142,8 +145,25 @@ function parse_configuration()
 function load_configuration()
 {
   parse_configuration "$KW_ETC_DIR/$CONFIG_FILENAME"
-  parse_configuration "$HOME/.kw/$CONFIG_FILENAME"
-  parse_configuration "$PWD/$CONFIG_FILENAME"
+  parse_configuration "$HOME/$KW_DIR/$CONFIG_FILENAME"
+
+  # New users may not have kworkflow.config at $PWD
+  if [[ -f "$PWD/$CONFIG_FILENAME" ]]; then
+    warning 'We will stop supporting kworkflow.config soon.'
+    if is_kernel_root "$PWD" &&
+      [[ $(ask_yN 'Do you want to update your configurations to .kw?') =~ 1 ]]; then
+      mkdir -p "$PWD/$KW_DIR/"
+      mv -i "$PWD/$CONFIG_FILENAME" "$PWD/$KW_DIR/$CONFIG_FILENAME"
```

Comment:
Why do we need `-i`? I don't think we need this since we already asked about the action in the `if` condition.